includes/admin/views/html-pay-order.php

```diff
@@ -0,0 +1,77 @@
+<p class="form-field form-field-wide">
+    <button class="button button-secondary wc-globalpayments-pay-order"><?php esc_html_e( 'Pay for Order', 'woo-payment-gateway' ); ?></button>
+	<?php echo wc_help_tip( __( 'Admins can process customer orders over the phone using this functionality.', 'woo-payment-gateway' ) ); ?>
+</p>
+<script type="text/template" id="tmpl-wc-globalpayments-pay-order-modal">
+    <div class="wc-backbone-modal">
+        <div class="wc-backbone-modal-content">
+            <section class="wc-backbone-modal-main" role="main">
+                <header class="wc-backbone-modal-header">
+                    <h1><?php esc_html_e( 'Pay for Order', 'globalpayments-gateway-provider-for-woocommerce' ); ?></h1>
+                    <button
+                            class="modal-close modal-close-link dashicons dashicons-no-alt">
+                        <span class="screen-reader-text">Close modal panel</span>
+                    </button>
+                </header>
+                <article>
+                    <form id="wc-globalpayments-pay-order-form">
+						<?php wp_nonce_field( 'woocommerce-globalpayments-pay', 'woocommerce-globalpayments-pay-nonce' ); ?>
```

Comment:
Some of the PHP code in here is not correctly indented (it has more tabs/spaces than it should)